gen-apidocs/generators/html.go

```diff
@@ -635,9 +577,9 @@ func (h *HTMLWriter) generateHTML(navContent string) error {
 	fmt.Fprintf(html, "<!DOCTYPE html>\n<HTML>\n<HEAD>\n<META charset=\"UTF-8\">\n")
 	fmt.Fprintf(html, "<TITLE>%s</TITLE>\n", h.TOC.Title)
 	fmt.Fprintf(html, "<LINK rel=\"shortcut icon\" href=\"favicon.ico\" type=\"image/vnd.microsoft.icon\">\n")
-	fmt.Fprintf(html, "<LINK rel=\"stylesheet\" href=\"/css/bootstrap-4.3.1.min.css\">\n")
-	fmt.Fprintf(html, "<LINK rel=\"stylesheet\" href=\"/css/fontawesome-4.7.0.min.css\" type=\"text/css\">\n")
-	fmt.Fprintf(html, "<LINK rel=\"stylesheet\" href=\"/css/style_apiref.css\" type=\"text/css\">\n")
+	fmt.Fprintf(html, "<LINK rel=\"stylesheet\" href=\"css/bootstrap-4.3.1.min.css\" type=\"text/css\">\n")
+	fmt.Fprintf(html, "<LINK rel=\"stylesheet\" href=\"css/fontawesome-4.7.0.min.css\" type=\"text/css\">\n")
+	fmt.Fprintf(html, "<LINK rel=\"stylesheet\" href=\"css/style_apiref.css\" type=\"text/css\">\n")
```

Comment:
These links are to ensure that the stylesheets can be found at k8s.io/docs website.

Reply:
Yeah, I had to modify them to make it easier to host the temporary files on my server. Before merging this, I will revert them :)

On that note, (how) do we need to coordinate between this repo and the website repo? Both PRs would need to be merged at roughly the same time, and then new API docs would need to be generated and copied over. What's the best way to do this cleanly?

Reply:
We fix the generator first, then we regenerate the reference and copy them over.